ICO_Template/ICO_Template.cs

```diff
@@ -141,7 +145,6 @@ public static bool Transfer(byte[] from, byte[] to, BigInteger value)
         {
             if (value <= 0) return false;
             if (!Runtime.CheckWitness(from)) return false;
-            if (from == to) return true;
```

Comment:
What is the main reason for removing this line?

Reply:
this is error remove, i will close this pull request, resubmit a new pull request.